headeredevent.go

```diff
@@ -26,6 +27,16 @@ type HeaderedEvent struct {
 	Event
 }
 
+// Unwrap extracts the event object from the headered event.
+func (e *HeaderedEvent) Unwrap() Event {
+	if e.RoomVersion == "" {
+		panic("gomatrixserverlib: malformed HeaderedEvent doesn't contain room version")
```

Comment:
Again, I thought you were morally opposed to panicking?

Reply:
If you intend to fix this later, then you need to be adding TODOs.

Reply:
Added a TODO for now. Ideally I don't want to panic long term, but while room versions are mid-work it is easier to highlight where things go wrong.